src/modules/home/components/LoggedInHome/LoggedInHome.tsx

```diff
@@ -271,7 +271,8 @@ const LoggedInHome: FC<DefaultSeoInfo> = ({ url }) => {
 						</p>
 					</section>
 
-					{(active?.items || []).length > 0 && (
+					{(active?.items.filter((visit) => !visit.id.includes('permanent-id')) || [])
```

Comment:
Create a reusable filter function since for this. This way, it'll be easier to make changes. 

Maybe add a comment why we need to do this :)